roles/openshift_inventory/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- no_log: True  # keep task descriptions legible
+  block:
+
+  - name: Standardize on latest variable names
+    set_fact:
+      # goal is to deprecate deployment_type in favor of openshift_deployment_type.
+      # both will be accepted for now, but code should refer to the new name.
+      # TODO: once this is well-documented, add deprecation notice if using old name.
+      deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+      openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+
+  - name: Normalize openshift_release
+    set_fact:
+      # normalize release if provided, e.g. "v3.5" => "3.5"
+      openshift_release: "{{ openshift_release | string | regex_replace('^v', '') }}"
+    when: openshift_release is defined
```

Comment:
Isn't `openshift_release` required to be defined? Is it unlike `openshift_deployment_type`?
If I had to guess, then I could imagine we figure out `openshift_release` based on available RPM/image versions, is that the case?